remote_execution/oss/re_grpc/src/client.rs

```diff
@@ -651,7 +716,11 @@ fn convert_action_result(action_result: ActionResult) -> anyhow::Result<TActionR
     Ok(action_result)
 }
 
-async fn download_impl<F, Fut>(request: DownloadRequest, f: F) -> anyhow::Result<DownloadResponse>
+async fn download_impl<F, Fut>(
```

Comment:
FWIW, I imagine downloads probably should get the same treatment as uploads (i.e. use the bytestream API)